int32 roboyID
int32 leg
int32 state